tf-modules/ingest/hello-world-task.tf

```diff
@@ -6,7 +6,7 @@ resource "aws_lambda_function" "hello_world_task" {
   role             = var.lambda_processing_role_arn
   runtime          = "nodejs10.x"
   timeout          = 300
-  memory_size      = 256
+  memory_size      = 512
```

Comment:
Why did the memory needs for our hello world task double?

Reply:
For the same reasons as https://github.com/nasa/cumulus/pull/1578 - upping memory updates processor power, the savings is .... roughly? 1:1. 

There's room for a discussion of tuning/risk profile rewards here, for sure.